runtime/op/meta/pruner.go

```diff
@@ -0,0 +1,27 @@
+package meta
+
+import (
+	"github.com/brimdata/zed"
+	"github.com/brimdata/zed/order"
+	"github.com/brimdata/zed/runtime/expr"
+)
+
+type Pruner struct {
```

Comment:
Nit: No need to export this.
```suggestion
type pruner struct {
```